docs/users-guide/visualizations/row-chart.md

```diff
@@ -0,0 +1,48 @@
+### Row charts
```

Comment:
The content on this page page should probably live in the line, bar, and area chart page.

Reply:
I integrated the row charts content into the line, bar, and area chart page but the organization of the pages was inconsistent with each other. So I deleted the "Row charts" header and "Histograms" header to make the layout more consistent with the page. Let me know if that's okay